data/helpers.d/utils

```diff
@@ -24,3 +24,19 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Curl abstraction to simplify its call
+#
+# Helpfully mostly for curl arguments
```

Comment:
This helper is helpful particularly for the arguments of curl.
-kL -H "Host: $domain" --resolve $domain:443:127.0.0.1

Like that, the packager doesn't be careful about which arguments it's better to use.